### DISCUSSION TITLE
Which stamps did I buy?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#793] [Galaxy Trivia] [answered]
Yesterday, I was asked to buy some stamps. Stamps are available in 2p, 7p, 10p, 15p and 20p denominations. For three types of stamp, I was asked to buy five of each. For the other two types of stamps, I was asked to buy six of each. Unfortunately, I forgot which I was supposed to buy five of and which to buy six of. Luckily I had been given the exact money required to buy the stamps, £3.00 and the shopkeeper was able to give me the correct stamps.

**Comment by peterbaccaro** [comment, marked as answer]
Five lots of 2p, 7p and 15p and six lots of 10p, and 20p.
Five lots of each of the stamps comes to £2.70.
So two more stamps 10p and 20p are required to get to £3.00.

**Reply by ParanoidUser** [reply]
Not sure how £2.70+10p and 20p relates to the answer, but I'll take it. 📧
